share/governance.md

```diff
@@ -0,0 +1,95 @@
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. Condorcet(https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
+
+*How to become one*: Win in the yearly election.
+
+
+
+Lazy Consensus
+--------------
+
+Lazy consensus is a very important concept within the project. It is this process that allows a large group of people to efficiently reach consensus, as someone with no objections to a proposal need not spend time stating their position, and others need not spend time reading such statements.
+
+For lazy consensus to be effective, it is necessary to allow at least 72 hours before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest and respond to the proposal. This time period is chosen so as to be as inclusive as possible of all participants, regardless of their location and time commitments.
+
+We do require one affirmative vote as part of the Lazy consensus model.
+
+Vox Pupuli has a public mailing list. This list is low-traffic and is for discussion of anything Vox Pupuli related. Controversial patches, nominations for promotion, policy shifts, and anything else can be brought to this list. All contributors and PMC members are expected to be on this list. Release announcements will go out to this list, but will have ``[Release]`` in the subject line so they can be easily filtered.
+
+
+
+
+PMC Removal
+-----------
+
+A member of the PMC can be removed before the end of their 1 year term by a supermajority (2/3) vote of the PMC. This can happen for a number of reasons, but the expected reason is following a Code of Conduct violation incident. A member of the PMC can step down at any time.
+
+If a vacancy is left on the PMC, the PMC can optionally instruct the elections officer to perform another election or wait until the next election cycle.
+
+
+Transparency
+------------
+
+Building community trust in the governance of an open-source project is vital to its success. To that end, decision making must be done in a transparent, open fashion. No decisions about the project’s direction, bug fixes or features may be done without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
+
+
+
+Election
+--------
+
+The details of the election process are up to the elections officer, subject to approval by the PMC. The users, collaborators, and contributors all get to vote in the election.
```

Comment:
I agree that it's not perfect. And it totally has the potential to get sketchy. That's why this person is selected by the PMC, not just some rando.

Like with the self-governace above, my idea here was to give power and flexibility to the Elections officer, precisely because encoding how to run an election is very hard to do. By empowering the elections officer, that person can make calls during the election to keep the process as smooth as possible. 

An example would be that the ballot was sent out but we forgot to say if the polls close at 12:00 UTC or 12:00 PST.. The elections officer can handle that situation by making a decision and sending out a follow up email, some people will probably be grouchy about the result, but no one is unclear about what happened and whether or not the elections officer had the authority to do what they did. This is way harder if the PMC has to get involved to ratify changes to the election/governance process, and harder still if the constituency needs to be consulted mid-election on the election process.

@vmbrasseur @igalic to your point, I would love to have a fully documented, fully automated election process that was approved by the constituency. But the reality is we're gonna have to run a few of these before we even know what those look like. I should know, I ran the first poll/election thing we did here and it was a bit of a shit-show. In the absence of knowing exactly how to do it, I think giving a trusted individual the authority to run it as they see fit is a good compromise. Once we know how this should be done, and we've failed(improved) a few times, we can go back and write it up and lock it down.

Another reason to leave election procedure out of this document is that we could debate and infinitely improve election procedures for possibly longer than the entire rest of the document, and I'd like to see us approve this and get on with it.